docs/source/api/search.rst

```diff
@@ -15,11 +15,13 @@ All search methods implement the :class:`SearchMethod` protocol.
 
    .. autoclass:: SearchMethod
       :members:
+      :special-members: __call___
```

Comment:
Strange, this doesn't work for the `search` functions but it does work for `StoppingCriterion`.